src/main/resources/add-ons-to-index.json

```diff
@@ -42,6 +42,27 @@
       "package": "openmrs-owa-cohortbuilder"
       }
     },
+    {
+      "uid": "org.openmrs.owa.obsadmin",
+      "type": "OWA",
+      "name": "Obsadmin",
```

Comment:
Can you change the name to Observations Administration?

Reply:
Alright